Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,42 @@
 import React from 'react';
 import './App.css';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay'
+import Button from './components/ButtonComponents/Button'
 
 const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
-  );
+    return (
```

Comment:
This return statement looks great with properly passing props into components and good className separations for the different rows and sections.